webpack/react_app/components/TargetingHosts/HostActions/HostActions.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, SplitButton, MenuItem } from 'patternfly-react';
+
+const HostActions = props => {
+  const { actions } = props;
+
+  if (actions === undefined || actions.length === 0) {
+    return '';
+  }
+
+  if (actions.length === 1) {
+    return <Button href={actions[0].path}>{actions[0].name}</Button>;
+  }
+  return (
+    <SplitButton href={actions[0].path} title={actions[0].name} id="">
+      {actions.slice(1, actions.length).map(link => (
+        <MenuItem key={link.path} href={link.path}>
```

Comment:
This can be switched with `/react_app/components/common/ActionButtons/ActionButtons.js`